recipes/tiberius/build.sh

```
@@ -4,4 +4,12 @@ mkdir -p ${PREFIX}/bin
 cp -R $SRC_DIR/bin/* ${PREFIX}/bin/
 chmod +x ${PREFIX}/bin/tiberius.py
 ln -s ${PREFIX}/bin/tiberius.py ${PREFIX}/bin/tiberius
-sed -i '1s|^|#!/usr/bin/env python3\n|' ${PREFIX}/bin/tiberius
\ No newline at end of file
+sed -i '1s|^|#!/usr/bin/env python3\n|' ${PREFIX}/bin/tiberius
+
+#Temporary: until the specific branch of learnMSA compatible with Tiberius is merged
+#Adding learnMSA
+git clone https://github.com/Gaius-Augustus/learnMSA
+cd learnMSA
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)